opencti-platform/opencti-graphql/src/domain/settings.js

```diff
@@ -197,3 +197,11 @@ export const getCriticalAlerts = async (context, user) => {
   // no alert
   return [];
 };
+
+export const getMaxMarkings = async (context, user, settings) => {
```

Comment:
suggestion to rename to `getDataSharingMaxMarkings` ? to be more explicit that it's the max markings setting for data sharing.